vitrina/structure/services.py

```diff
@@ -858,14 +858,14 @@ def get_data_from_spinta(model: Union[Model, str], uuid: str = None, query: str
         url = f"https://get.data.gov.lt/{model}/?{query}"
     try:
         res = requests.get(url)
-    except BaseException:
-        return {}
+    except BaseException as e:
```

Comment:
Negali būti naudojamas BaseException, reikia naudoti konkrečias klaidų klases, šiuo atveju RequestException: https://docs.python-requests.org/en/latest/user/quickstart/#errors-and-exceptions